CodeStarterCamp_Week4/step01.swift

```diff
@@ -32,45 +65,39 @@ class BodyCondition {
         self.muscularEndurance = muscularEndurance
         self.degreeOfFatigue = degreeOfFatigue
     }
-    
+}
+
+func startTraining (_ activity: Activity, _ bodyCondition: BodyCondition) {
+    activity.action(bodyCondition)
 }
```

Comment:
요 함수는 반드시 필요한 함수일까요?
요 함수를 정의하신 이유가 무엇일까요? 🤔

별개로 띄어쓰기 오타가 있네요 😅
```suggestion
func startTraining(_ activity: Activity, _ bodyCondition: BodyCondition) {
    activity.action(bodyCondition)
}
```

Reply:
그렇지 않아도 이 부분 관련해서 수정을 하고 있습니다. startRoutine 한 곳에서 너무 많은 것을 처리하려고 하다보니 굳이 만들지 않아도 될 함수까지 만들어버린것 같아서 다시 보고 있습니다. 수정해보겠습니다!!